mikeio/dataset.py

```diff
@@ -543,6 +565,95 @@ def to_dataframe(self, unit_in_name=False):
     def _ipython_key_completions_(self):
         return [x.name for x in self.items]
 
+    @staticmethod
+    def create_empty_data(n_items=1, n_timesteps=1, n_elements=None, shape=None):
```

Comment:
```suggestion
    def empty(n_items=1, n_timesteps=1, n_elements=None, shape=None):
```

Reply:
empty_data() ? It's not returning an empty Dataset,,,